src/dune_engine/clflags.mli

```diff
@@ -53,3 +53,11 @@ val promote_install_files : bool ref
 
 (** Whether we are ignoring rules with [(mode promote)] *)
 val ignore_promoted_rules : bool ref
+
+type on_missing_dune_project_file =
+  | Error
+  | Warn
+  | Ignore
+
+(** Desired behavior when dune project file is absent *)
+val dune_project_file : on_missing_dune_project_file ref
```

Comment:
The rename I suggested earlier was specifically driven by this name, since it's completely non-obvious at the call sites what `Clflags.dune_project_file` means.

Reply:
renamed the `val ..` as well

Reply:
Thanks!